java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -92,8 +95,14 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
 
         baseTransaction.configResetStat();
 
-        // TODO
-        // baseTransaction.pfadd(hllKey1, new String[0]);
+        baseTransaction
+                .pfadd(hllKey1, new String[0])
```

Comment:
we don't need to test the zero case.

Reply:
It is already reverted in https://github.com/Bit-Quill/glide-for-redis/pull/166